command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
+rucio account attribute list
+rucio account attribute add
+rucio account attribute remove
+rucio account limit list
+rucio account limit add
+rucio account limit remove
+rucio account identity list
+rucio account identity add
+rucio account identity remove
+```
+
+# Config 
+```
+rucio config list
+rucio config add
+rucio config remove
+```
+
+# DID 
+```
+rucio did list
+rucio did show
+rucio did remove
+rucio did touch
+rucio did parent
+rucio did attach
+rucio did detach
+rucio did upload
+rucio did download
+rucio did content list
+rucio did content history
+rucio did content close
+rucio did content open
+rucio did metadata list
+rucio did metadata add
+rucio did metadata remove
+rucio did container add
+rucio did dataset add
+```
+
+# Replica 
+```
+rucio replica list
+rucio replica dataset
+rucio replica pfn
+rucio replica tombstone
+rucio replica state suspicious
+rucio replica state quarantine
+rucio replica state bad
+rucio replica state temp-unavailable
+```
+
+# RSE
+```
+rucio rse list
+rucio rse show
+rucio rse remove
+rucio rse add
+rucio rse update
+rucio rse attribute list
+rucio rse attribute add
+rucio rse attribute remove
+rucio rse distance list
+rucio rse distance add
+rucio rse distance remove
+rucio rse distance set
+rucio rse protocol add
+rucio rse protocol remove
+rucio rse limit add
+rucio rse limit remove
+rucio rse qos list
+rucio rse qos add
+rucio rse qos remove
+
+```
```

Comment:
One the whole, the RSE-related commands are well organised. I will repeat my comment about each group having the five basic subcommands.

 The only existing item that doesn’t fit perfectly is `rse show`. Following the entity it should only return the RSE settings, but that typically is not useful.

The area that I believe is currently missing is RSE usage counters.

Reply:
rse usage is bundled in with `rse show`. It's mapped to `info_rse` which shows, usage, limits, and attributes per rse.